--- widelands.orig/xdg/org.widelands.Widelands.appdata.xml.stub
+++ widelands/xdg/org.widelands.Widelands.appdata.xml.stub
@@ -29,13 +29,13 @@
   </screenshots>
   <update_contact>widelands-public_at_lists.sourceforge.net</update_contact>
   <provides>
-    <binary>widelands</binary>
     <id>widelands.desktop</id>
     <id>org.widelands.widelands.desktop</id>
     <id>org.widelands.widelands</id>
   </provides>
   <launchable type="desktop-id">org.widelands.Widelands.desktop</launchable>
   <releases>
+    <release date="2021-12-04" version="1.1~git25545[4bcc4e3@master]"/>
     <release date="2021-06-15" version="1.0"/>
     <release date="2020-07-18" version="~Build 21"/>
     <release date="2019-05-02" version="~Build 20"/>
