--- widelands.orig/utils/update_appdata.py
+++ widelands/utils/update_appdata.py
@@ -5,6 +5,7 @@
 import codecs
 import json
 import os.path
+import shutil
 import sys
 
 # This script collects translations for the appdata.xml and .desktop files
@@ -179,7 +180,12 @@
 
 
 # Validata Appdata
-subprocess.run(['appstreamcli', 'validate', appdata_filepath])
+if shutil.which('appstreamcli'):
+    subprocess.run(['appstreamcli', 'validate', appdata_filepath])
+elif shutil.which('appstream-util'):
+    subprocess.run(['appstream-util', 'validate-relax', appdata_filepath])
+else:
+    print('Cannot validate generated appdata file.')
 
 # Validate desktop file. We don't get return codes, so we have to parse it
 desktop_result = subprocess.run(['desktop-file-validate', desktop_filepath],
