--- widelands.orig/utils/update_appdata.py
+++ widelands/utils/update_appdata.py
@@ -179,7 +179,7 @@
 
 
 # Validata Appdata
-subprocess.run(['appstreamcli', 'validate', appdata_filepath])
+subprocess.run(['appstream-util', 'validate-relax', appdata_filepath])
 
 # Validate desktop file. We don't get return codes, so we have to parse it
 desktop_result = subprocess.run(['desktop-file-validate', desktop_filepath],
